embed/ephy-embed.c

```diff
@@ -957,3 +958,28 @@ ephy_embed_inspector_is_loaded (EphyEmbed *embed)
 
   return embed->inspector_loaded;
 }
+
+void
+ephy_embed_attach_notification_manager (EphyEmbed *embed)
+{
+  EphyNotificationManager *manager;
+
+  g_return_if_fail (EPHY_IS_EMBED (embed));
+
+  manager = ephy_notification_manager_dup_singleton ();
```

Comment:
Hm, I will comment more on this when I get to the EphyNotificationManager file, but we'll want to change this. Duplicating a singleton object doesn't make any sense. Should it be named `ephy_notification_manager_get_default()`?